.github/release-drafter.yml

```diff
@@ -0,0 +1,47 @@
+name-template: 'v$RESOLVED_VERSION'
+
+tag-template: 'v$RESOLVED_VERSION'
+
+version-template: '$MAJOR.$MINOR.$PATCH'
+
+categories:
+  - title: '🚀 Features'
+    labels:
```

Comment:
Yeah, Release Drafter will categorize them based on labeled PRs. There is an option to include issues for the Release Drafter. For example we can add the following under `on:` in `workflows/release-drafter.yml`

```yml
  issues:
    types: [closed]
```

This will then trigger the Release Drafter to run when issues are closed, but it's effects are not documented. We could try it and see what Release Draft it spits out.